release-notes/release-notes55.md

```diff
@@ -15,79 +15,196 @@ These release notes provide an overview of the changes for {{site.productname}}
+The CSS files bundled with {{site.productname}} have been updated to better support the styling options available in the `table` dialog.
 
-For information on the new persistent toolbar option, see: [User interface options - `toolbar_persist`]({{site.baseurl}}/configure/editor-appearance/#toolbar_persist).
+- For information on the bundled CSS files, see: [Customizing the editable area with `content_css`]({{site.baseurl}}/general-configuration-guide/customize-ui/#customizingtheeditableareawithcontent_css).
+- To view the CSS files, see: [tinymce-dist GitHub Repository - Content CSS files](https://github.com/tinymce/tinymce-dist/tree/master/skins/content).
 
 ### New listbox component for custom dialogs
 
 The new listbox component for custom dialogs is a composite component with a label and a dropdown list of options for users to select from. This component displays a menu-like dropdown and allows nested options to be shown to users.
 
 For information on the listbox dialog component, see: [Dialog components - listbox]({{site.baseurl}}/ui-components/dialogcomponents/#listbox).
 
-### The toolbar drawer can now be controlled using commands
+### New `hasPlugin` for detecting the availability of plugins
 
-A new editor command (`execCommand('ToggleToolbarDrawer')`) and editor state query (`queryCommandState('ToggleToolbarDrawer')`) have been added. The command allows the Toolbar Drawer (which holds excess toolbar buttons) to be opened and closed using editor commands. The current state of the Toolbar Drawer can also be queried.
+The new `hasPlugin` API method can be used to detect if a plugin is present in the editor configuration or if the plugin has been loaded. This API is useful if custom functionality such as a custom plugin, is dependent on the APIs of a plugin.
 
-For information on:
+For information on the `editor.hasPlugin()` API, see: [tinymce.Editor APIs - `hasPlugin`]({{site.baseurl}}/api/tinymce/tinymce.editor/#hasplugin).
 
-- The Toolbar Drawer, see: [User interface options - `toolbar_mode`]({{site.baseurl}}/configure/editor-appearance/#toolbar_mode).
-- Commands and query commands available in {{site.productname}}, see: [Commands Available for TinyMCE]({{site.baseurl}}/advanced/editor-command-identifiers/).
+### New StyleSheetLoader `unload` and `unloadAll` APIs
 
-### New line height support in the editor
+Two new APIs have been for unloading CSS files:
 
-{{site.productname}} 5.5 now supports adjustable line heights. There is a new `lineheight` button available for toolbars, and a new `lineheight` menu item (included by default in the format menu). There is also a new `LineHeight` editor command to change line height and a `LineHeight` query command value to get the current line height.
+- `unload` - Unloads the specified CSS file if it is not in use.
+- `unloadAll` - Unloads all CSS files not in use.
```

Comment:
This is incorrect. It takes a list of URLs unloads the ones not in use (basically the same as `unload`, except it takes multiple URLs).

Reply:
```suggestion
- `unloadAll` - Accepts a list of URLs and unloads any CSS files not in use.
```
@lnewson ?

Reply:
LGTM :+1: